### PR TITLE
Add minor improvements

### PR DESCRIPTION
**Pull request by arnas**

### CODE REVIEW

File: omnisend/class-omnisend-core-bootstrap.php [hunk 1]
```diff
@@ -37,20 +37,25 @@
 // Omnisend for Woo plugin.
 const OMNISEND_CORE_WOOCOMMERCE_PLUGIN_API_KEY_OPTION = 'omnisend_api_key';
 
+
```
Comment (nerijuszaniauskas) [inline omnisend/class-omnisend-core-bootstrap.php:40]: Not needed.

File: omnisend/class-omnisend-core-bootstrap.php [hunk 2]
```diff
@@ -37,20 +37,25 @@
...
+
 spl_autoload_register( array( 'Omnisend_Core_Bootstrap', 'autoloader' ) );
 register_uninstall_hook( __FILE__, 'Omnisend_Core_Bootstrap::uninstall' );
 add_action( 'plugins_loaded', 'Omnisend_Core_Bootstrap::load' );
 
 class Omnisend_Core_Bootstrap {
 
 
+
```
Comment (nerijuszaniauskas) [inline omnisend/class-omnisend-core-bootstrap.php:49]: Same here, not needed. You can even delete one empty line

File: omnisend/class-omnisend-core-bootstrap.php [hunk 3]
```diff
@@ -93,10 +132,34 @@ public static function cron_schedules( $schedules ) {
...
+		};
+
+		$last_visit_time = Options::get_landing_page_last_visit_time();
+
+		return ! Options::is_landing_page_visited() && $has_seven_days_passed( $last_visit_time );
+	}
+
 	public static function load_omnisend_admin_styles(): void {
-		// phpcs:disable WordPress.Security.NonceVerification
+		// // phpcs:disable WordPress.Security.NonceVerification
```
Comment (nerijuszaniauskas) [inline omnisend/class-omnisend-core-bootstrap.php:161]: This is not needed.

**Review by nerijuszaniauskas — DISMISSED** [review]

File: omnisend/class-omnisend-core-bootstrap.php (hunk 1 above)
Comment (arnas) [reply omnisend/class-omnisend-core-bootstrap.php:41]: The lint allows empty line

File: omnisend/class-omnisend-core-bootstrap.php [hunk 4]
```diff
@@ -129,26 +179,52 @@ public static function cron_schedules( $schedules ) {
...
+			plugin_dir_url( __FILE__ ) . 'styles/site-styles.css?' . time(),
+			array(),
+			OMNISEND_CORE_PLUGIN_VERSION,
+		);
+	}
+
+	public static function load_omnisend_site_styles(): void {
+		wp_enqueue_style(
+			'site-styles.css',
+			plugin_dir_url( __FILE__ ) . 'styles/site-styles.css?' . time(),
```
Comment (Gabiermi) [inline omnisend/class-omnisend-core-bootstrap.php:223]: Why /site-styles inserted twice?

File: omnisend/class-omnisend-core-bootstrap.php [hunk 5]
```diff
@@ -74,12 +82,14 @@ function ( $user_login, $user ) {
 
 			add_action( OMNISEND_CORE_CRON_SYNC_CONTACT, 'Omnisend\Internal\Sync::sync_contacts' );
 		}
+
+		// self::migrate_options(); fix bug and uncomment
```
Comment (rokeras) [inline omnisend/class-omnisend-core-bootstrap.php:86]: relevant?

File: omnisend/class-omnisend-core-bootstrap.php (hunk 4 above)
Comment (arnas) [reply omnisend/class-omnisend-core-bootstrap.php:223]: one time for admin, one time for website styles ( when wp admin bar is visible in admin)

File: omnisend/class-omnisend-core-bootstrap.php (hunk 5 above)
Comment (arnas) [reply omnisend/class-omnisend-core-bootstrap.php:86]: yes, it will be need to updated when we will add 7 day notification option

**Review by rokeras — APPROVED** [review]

**Pull request merged**